.code-samples.meilisearch.yaml

```diff
@@ -18,7 +18,7 @@ delete_documents_1: |-
 search_post_1: |-
 get_update_1: |-
 get_all_updates_1: |-
-  var updates = await client.Index("movies").GetAllUpdateStatus();
+  var updates = await client.IndexAsync("movies").GetAllUpdateStatus();
```

Comment:
`IndexAdync` does not exist
`GetAllUpdateStatus` should be `GetAllUpdateStatusAsync`

Reply:
Sorry, I was sure I had fixed these lines. Should be good now.